web-client/iron-svelte-client/src/services/server-bridge.service.ts

```diff
@@ -1,14 +1,13 @@
 import type { Observable } from 'rxjs';
 
 export interface ServerRect {
-    free(): void;
-
     clone_buffer(): Uint8Array;
 
     bottom: number;
     left: number;
     right: number;
     top: number;
+    free(): void;
```

Comment:
style: I’m not sure what is the convention here, but the properties are now in-between two functions :thinking:

Reply:
My bad. Let me revert

Reply:
Reverted